js/battle.js

```diff
@@ -6467,11 +6467,13 @@ var Battle = (function () {
 			poke.removeVolatile('typeadd');
 			poke.removeVolatile('typechange');
 
-			var newSpecies;
-			var commaIndex = args[2].indexOf(',');
-			if (commaIndex === -1) {
-				newSpecies = args[2];
-			} else {
+			var newSpecies = args[2];
+			var commaIndex = newSpecies.indexOf(',');
+			if (commaIndex !== -1) {
+				var level = $.trim(newSpecies.substr(commaIndex + 1));
+				if (level.charAt(0) === 'L') {
+					poke.level = +level.substr(1);
```

Comment:
...this is `NaN` if the Pokémon has a gender.

Reply:
![image](https://user-images.githubusercontent.com/551184/30107160-ee9e3faa-92cb-11e7-92d4-441ca269e3dc.png)